skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +474,21 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        rpix, phipix = get_r_phi(shape, origin)
```

Comment:
I tried your two suggestions by adding these two lines at the bottom of get_r_phi:

    rpix = radial_grid((rowc,colc), (shape[0],shape[1])).T
    phipix = angle_grid((rowc,colc), (shape[0],shape[1])).T

The self-tests then failed, I think because the shapes are flipped (vs. the original) in the above call.

I then tried your other suggestion by adding two lines instead:

    rpix = radial_grid((colc, rowc), (shape[0], shape[1]))
    phipix = angle_grid((colc, rowc), (shape[0], shape[1]))

But the self-tests also failed, although it's not clear to me why.  If I left out the second "phipix" line, the self-tests would pass.

When I first implemented this I remember playing around with mesh grid for a couple of hours to see if I could avoid the indexing='ij' option, but failed.  But perhaps someone smarter than me knows of a way to do it.